### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by levy002**
My Portfolio milestone 2: My recent wok

In this milestone I :

   *  Added lists of projects
   *  Styling the work (projects) section and making it responsive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -94,11 +94,151 @@ <h2>Software Developer</h2>
...
+      <!-- Work -->
+
+      <section class="work">
+        <div id="work-title">
+          <h2>MY RECENT <span id="break">WORK</span></h2>
+          <img src="/images/vector-line.png" alt="line" />
+        </div>
+
+        <ul class="all_projects">
+          <li class="project">
```
Comment (ReshyResh) [inline index.html:110]: - [ ] You did a great job displaying the different works section using `display:grid`, however, as per requirement, each single work card should have its content displayed using Flexbox.
Hint: use `display:flex` with a flex-flow of column.

File: styles.css [hunk 2]
```diff
@@ -135,11 +168,110 @@ li > * {
...
+  grid-gap: 5px;
+}
+
+.project-body p {
+  font-family: sans-serif, Inter;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 28px;
+  color: #dcdce8;
+}
```
Comment (ReshyResh) [inline styles.css:235]: - [ ] Please make sure to center the elements as in the template. 
Currently, your cards have the image and button not centered, try fixing that to match the template.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @levy002 , 

You did a really good good job so far, almost nailing it on the first attempt!
There are some issues that you still need to work on to go to the next project but you are almost there!

To highlight:
  - ✔Descriptive commit messages.
  - ✔Design matches template requirements.
  - ✔Used `grid` correctly to display the work cards.


## Required Changes ♻️
_Please check the comments under the review._

----

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -118,13 +151,14 @@ li > * {
 }
 
 /*
-  SUPPORTING ITEMS & SOCIAL MEDIA
+  SUPPORTING ITEMS & SOCIAL MEDIAs
 */
```
Comment (aliabbani) [inline styles.css:155]: [OPTIONAL] Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.
**Kindly remove all commented code in your style.css file 😊 .**

File: index.html [hunk 4]
```diff
@@ -94,11 +94,151 @@ <h2>Software Developer</h2>
...
         </ul>
 
-        <div class="work">
+        <div class="work-direction">
           <img src="/images/Arrow.png" alt="down-arrow" />
           <span>Work</span>
         </div>
       </div>
+
+      <!-- Work -->
```
Comment (aliabbani) [inline index.html]: [OPTIONAL] Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.
**Kindly remove all commented code in your index.html file 😊 .**

**Review by aliabbani — APPROVED** [review]
Hi @levy002  ✋, 

APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  🎉  🎉  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**